source/ShopPanel.cpp

```diff
@@ -388,6 +390,21 @@ bool ShopPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, boo
 		return SetScrollToTop();
 	else if(key == SDLK_END)
 		return SetScrollToBottom();
+	else if((key == 'k' || (key == 'p' && (mod & KMOD_SHIFT))))
+	{
+		bool allParked = true;
+		const Ship *flagship = player.Flagship();
+		for(Ship *ship : playerShips)
+		{
+			if(!ship->IsDisabled() && ship != flagship)
```

Comment:
Can you have disabled ships in the shop?

Reply:
No, they don't show up when in system but disabled. I think this was there originally just for defensive purposes (there's a lot of "this shouldn't happen but just in case…" code).
It was there originally so I left it.